components/header/index.styl

```diff
@@ -1,77 +1,106 @@
+header
+  height 50vh
+  position relative
+
+.slide-show-container
+  position relative
+  height 100%
+
+.header-slide
+  width 100%
+  height 100%
+  background-position center center
+  background-size cover
+  -webkit-transition: opacity 1s ease-in
+  -moz-transition opacity 1s ease-in
+  -o-transition opacity 1s ease-in
+  -ms-transition opacity 1s ease-in
+  transition opacity 1s ease-in
+  position absolute
+  top 0
+  background-image url('./images/banner-3.jpg')
+  opacity 0
+
+.active-banner
+  opacity 1
+  z-index 500
+
+.slide-1
+  background-image url('./images/banner-1.jpg')
+
+.slide-2
+  background-image url('./images/banner-2.jpg')
+
+.slide-3
+  background-image url('./images/banner-3.jpg')
+
+.header-content
+  height 100%
+  display flex
+  flex-direction column
+  justify-content space-around
+  z-index 501
+  position absolute
+  top 0
+
+.header-text
+  width 85%
+  margin 10% auto
+  padding-top 30px !important
```

Comment:
We talked offline about the better way to vertically position this guy (display table method).